.gitignore

```diff
@@ -0,0 +1 @@
+.env
```

Comment:
This gitignore file is not complete. We should not be uploading the node modules dependencies. Only the package.json file is necessary.